source/includes/openstack/_routerInterfaces.md

```diff
@@ -0,0 +1,111 @@
+### Router Interfaces
+
+Router interfaces connect a router to a network.
+
+#### List router interfaces
+
+```shell
+curl -H "MC-Api-Key: your_api_key" \
+    "https://api.your.cloudmc/v1/services/compute-os/devel/routerinterfaces?routerid=3de4f523-06ce-4955-acb0-b8e3d61ec582"
+```
+```json
+{
+	"data": [
+		{
+			"id": "b5f0cbbf-45f3-4e77-8a5c-90351f83e5f9",
+			"networkName": "test2",
+			"networkId": "d507486c-eacb-48ff-841e-e3938213d95f",
+			"portId": "b5f0cbbf-45f3-4e77-8a5c-90351f83e5f9",
+			"privateIp": "10.182.15.1",
+			"routerId": "3de4f523-06ce-4955-acb0-b8e3d61ec582"
+		}
+	],
+	"metadata": {
+		"recordCount": 1
+	}
+}
+```
+
+<code>GET /services/<a href="#service-connections">:service_code</a>/<a href="#environments">:environment_name</a>/routerinterfaces?routerid=<a href="#routers">:routerid</a></code>
+
+Retrieve a list of router interfaces associated with a router in an OpenStack environment.
+
+ Attributes                    | Description                         
+ ----------------------------- | -----------------------------------
+ `id`<br/>*UUID*               | The router interface's ID                     
+ `networkName`<br/>*string*    | The name of the connected network                   
+ `networkId`<br/>*UUID*        | The ID of the connected network   
+ `portId`<br/>*UUID*           | The ID of the port the interface is connected to (same as interface ID)
+ `privateIp`<br/>*string*      | The private IP of the router interface  
+ `routerId`<br/>*UUID*         | The ID of the parent router
+
+#### Retrieve a router interface
+
+```shell
+curl -H "MC-Api-Key: your_api_key" \
+    "https://api.your.cloudmc/v1/services/compute-os/devel/routerinterfaces/b5f0cbbf-45f3-4e77-8a5c-90351f83e5f9"
+```
+```json
+{
+	"data": {
+		"id": "b5f0cbbf-45f3-4e77-8a5c-90351f83e5f9",
+		"networkName": "test2",
+		"networkId": "d507486c-eacb-48ff-841e-e3938213d95f",
+		"portId": "b5f0cbbf-45f3-4e77-8a5c-90351f83e5f9",
+		"privateIp": "10.182.15.1",
+		"routerId": "3de4f523-06ce-4955-acb0-b8e3d61ec582"
+	}
+}
+```
+
+<code>GET /services/<a href="#service-connections">:service_code</a>/<a href="#environments">:environment_name</a>/routerinterfaces/:id</code>
+
+Retrieve a single router interface in an OpenStack environment.
+
+Attributes                    | Description                         
+----------------------------- | -----------------------------------
+`id`<br/>*UUID*               | The router interface's ID                     
+`networkName`<br/>*string*    | The name of the connected network                   
+`networkId`<br/>*UUID*        | The ID of the connected network   
+`portId`<br/>*UUID*           | The ID of the port the interface is connected to (same as interface ID)
```

Comment:
The router interfaces are basically ports. When the interface is created in open stack, the ID for the router interface is actually the ID of the port being connected to. We just have both id and portId to make it clear what the router interface identifier is.